src/guides/wallet-service/wallet-service.md

```diff
@@ -175,5 +175,70 @@ Added https options. Example:
 
 Notice that you can also specify which port your Wallet Service will run on (default is 3232).
 
+### Using free TLD and free SSL certificate to allow SSL/TLS support from mobile clients
+Self-signed certificates do not work very well, so using a TLD domain along with a root CA signed SSL certificate is the best method to ensure you can connect.
+
+First, get a free TLD from [http://www.dot.tk/](http://www.dot.tk/) and set the DNS to point to your server IP address. (Must be renewed every 12 months.)
+ (Be carful when signing up using an email address, dot.tk stores plaintext passwords, so try to use Facebook/Windows Live login if possible)
+
+Once you get your TLD and it is pointing to your IP, make sure you have Apache installed. (This is merely to make the SSL cert process automated, not required)
+
+```bash
+sudo apt-get update
+sudo apt-get install apache2
```

Comment:
This is to streamline the process.

Currently letsencrypt works best with apache, as it only requires one command to finish the whole process.

You can even set it to reroute all http traffic to https within that single command.

However, it is not necessary.

It would just take a really long time to explain the manual way of doing the challenge/response on any arbitrary piece of software.

I would rather assume that people looking to spin their own bws don't have a server or don't know much about servers to begin with. Anyone who owns a server to begin with probably already knows how to use letsencrypt manually, or at least knows how to google it.